operators/src/processing/meteosat/radiance.rs

```diff
@@ -0,0 +1,502 @@
+use crate::engine::{
+    ExecutionContext, InitializedRasterOperator, Operator, QueryContext, QueryProcessor,
+    RasterOperator, RasterQueryProcessor, RasterQueryRectangle, RasterResultDescriptor,
+    SingleRasterSource, TypedRasterQueryProcessor,
+};
+use crate::util::Result;
+use futures::stream::BoxStream;
+use futures::StreamExt;
+use geoengine_datatypes::primitives::SpatialPartition2D;
+use geoengine_datatypes::raster::{
+    EmptyGrid, Grid2D, GridShapeAccess, Pixel, RasterDataType, RasterPropertiesKey, RasterTile2D,
+};
+use num_traits::AsPrimitive;
+use std::convert::TryFrom;
+
+use async_trait::async_trait;
+use serde::{Deserialize, Serialize};
+
+//TODO Check
+type PixelOut = f32;
+use RasterDataType::F32 as RasterOut;
+use TypedRasterQueryProcessor::F32 as QueryProcessorOut;
+
+//TODO: For NAN the tests fail due to == comparisons
+const OUT_NO_DATA_VALUE: PixelOut = PixelOut::NEG_INFINITY;
+
+#[derive(Debug, Serialize, Deserialize, Clone, Copy)]
+#[serde(rename_all = "camelCase")]
+pub struct RadianceParams {}
+
+#[derive(Debug, Serialize, Deserialize, Clone, Copy)]
+pub struct RadianceState {
+    //TODO: Could be removed, if no data is a constant
+    out_no_data_value: PixelOut,
+}
+
+pub type Radiance = Operator<RadianceParams, SingleRasterSource>;
+
+pub struct InitializedRadiance {
+    result_descriptor: RasterResultDescriptor,
+    source: Box<dyn InitializedRasterOperator>,
+    state: RadianceState,
+}
+
+#[typetag::serde]
+#[async_trait]
+impl RasterOperator for Radiance {
+    async fn initialize(
+        self: Box<Self>,
+        context: &dyn ExecutionContext,
+    ) -> Result<Box<dyn InitializedRasterOperator>> {
+        let input = self.sources.raster.initialize(context).await?;
+
+        let in_desc = input.result_descriptor();
+
+        let out_desc = RasterResultDescriptor {
+            spatial_reference: in_desc.spatial_reference,
+            data_type: RasterOut,
+            // TODO: Is this correct?
+            measurement: in_desc.measurement.clone(),
```

Comment:
I not sure, referring to https://github.com/umr-dbs/mapping-core/blob/1651e1c1bc887214d9f525536f6bdf79a8c464de/src/operators/processing/meteosat/radiance.cpp#L82 it is `"radiance", "W·m^(-2)·sr^(-1)·cm^(-1)"`

Reply:
Fixed in: 3be3fd383500c6069bb49ffde1523b83a9c4b184